root/defaults/authentik_location.conf

```
@@ -0,0 +1,24 @@
+    # all requests to /outpost.goauthentik.io must be accessible without authentication. Change proxy_pass if needed. This is the default container name.
+    location /outpost.goauthentik.io {
+        proxy_pass          http://authentik_server_1:9000/outpost.goauthentik.io;
```

```suggestion
        include /config/nginx/resolver.conf;
        set $upstream_authentik authentik_server_1;
        proxy_pass http://$upstream_authentik:9000/outpost.goauthentik.io;
```
This would be more similar to how we have Authelia, and our reverse proxy confs. The resolver conf is probably significant to allow proper resolution of the container.